lib/hdiutil.js

```diff
@@ -2,10 +2,10 @@ var fs = require('fs')
 var temp = require('fs-temp')
 var util = require('./util')
 
-exports.convert = function (source, target, cb) {
+exports.convert = function (source, format, target, cb) {
   var args = []
   args.push('convert', source)
-  args.push('-format', 'UDZO')
+  args.push('-format', format)
   args.push('-imagekey', 'zlib-level=9')
```

Comment:
valid values are `['UDBZ', 'UDZO', 'NONE']` and UDZO is the default that is sent in by appdmg.js line 393. IIRC, there aren't any other current spots in the codebase that call this convert function, but if any are added they should specify UDZO as a default. Alternatively this function could specify that default, but it seemed to fit better in appdmg.js